vernac/declare.mli

```diff
@@ -206,16 +206,12 @@ exception AlreadyDeclared of (string option * Id.t)
 
 module Internal : sig
 
-  val map_entry_body : f:('a Entries.proof_output -> 'b Entries.proof_output) -> 'a proof_entry -> 'b proof_entry
-  val map_entry_type : f:(Constr.t option -> Constr.t option) -> 'a proof_entry -> 'a proof_entry
-  (* Overriding opacity is indeed really hacky *)
-  val set_opacity : opaque:bool -> 'a proof_entry -> 'a proof_entry
+   val map_entry_body : f:('a Entries.proof_output -> 'b Entries.proof_output) -> 'a proof_entry -> 'b proof_entry
+   val map_entry_type : f:(Constr.t option -> Constr.t option) -> 'a proof_entry -> 'a proof_entry
+   (* Overriding opacity is indeed really hacky *)
+   val set_opacity : opaque:bool -> 'a proof_entry -> 'a proof_entry
```

Comment:
Spurious change?

Reply:
Yes, sorry. Fixed. [originally due to a very different internal removal strategy]